static/app/views/detectors/detail.tsx

```diff
@@ -36,9 +38,14 @@ const priorities: Priority[] = [
 export default function DetectorDetail() {
   const organization = useOrganization();
   useWorkflowEngineFeatureGate({redirect: true});
+  const {detectorId} = useParams();
+  if (!detectorId) {
+    throw new Error(`Unable to find detector.`);
+  }
+  const {data: detector} = useDetectorQuery(detectorId);
```

Comment:
shouldn't be possible to not have detectorId
```suggestion
  const {detectorId} = useParams<{detectorId: string}>();
  const {data: detector} = useDetectorQuery(detectorId);
```